dataflux_pytorch/benchmark/checkpointing/singlenode/README.md

```diff
@@ -158,11 +154,35 @@ Dataflux's implementation of CheckpointIO for PyTorch Lightning is undergoing ac
    </td>
    <td style="background-color: #f3f3f3">1000
    </td>
-   <td style="background-color: #f3f3f3">2500
+   <td style="background-color: #f3f3f3">2,634
```

Comment:
Why are the file sizes different now? Let's also be consistent with whether or not we're using commas, here you're using one but in the 1000 number above we're not

Reply:
Updated to include commas everywhere